Northstar.Client/mod.json

```diff
@@ -31,6 +31,11 @@
 		{
 			"Name": "filter_map_hide_locked",
 			"DefaultValue": "0"
+		},
+		{
+			"Name": "modlist_hide_cv",
```

Comment:
no need to shorten convar, `modlist_hide_convar` instantly tells you where it is used and what it does